ironfish-cli/src/commands/deposit-all.ts

```diff
@@ -227,13 +227,13 @@ export default class DepositAll extends IronfishCommand {
       const sendableOre = confirmedBalance - fee
 
       // terminate condition
-      if (terminate && pendingBalance < sendableOre + fee) {
```

Comment:
sendableOre + fee = confirmedBalance due to the upper logic, if pendingBalance < confirmedBalance didn't make sense.
if pendingBalance < fee then destory, it's correct.